packages/backend-api/build.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="backend_api" default="list">
+    <property name="path.openssl.executable" value="openssl"/>
+    <property name="path.defuse-key-generate.executable" value="${path.bin}/generate-defuse-key"/>
+
+    <target name="backend-api-uninstall" description="Uninstall backend API from the project" hidden="true">
+        <exec executable="${path.backend-api}/install/uninstall.sh" checkreturn="true" passthru="true">
+            <arg value="monorepo"/>
```

Comment:
I didn't know that the `backend-api-(un)install` (originally `monorepo-backend-api-(un)install`) targets were defined here. As this build.xml gets included even in project repositories, the argument should be defined via property overriden in monorepo's `build.xml`:

```xml
<project name="backend_api" default="list">
    <property name="path.openssl.executable" value="openssl"/>
    <property name="path.defuse-key-generate.executable" value="${path.bin}/generate-defuse-key"/>
    <property name="backend-api.install-sh.arg" value=""/>

    <target name="backend-api-uninstall" description="Uninstall backend API from the project" hidden="true">
        <exec executable="${path.backend-api}/install/uninstall.sh" checkreturn="true" passthru="true">
            <arg value="${backend-api.install-sh.arg}"/>
         </exec>
    </target>
```

```xml
<?xml version="1.0" encoding="UTF-8"?>
<project name="Shopsys Framework" default="list">

    <property name="path.root" value="${project.basedir}/project-base"/>
    <property name="path.vendor" value="${project.basedir}/vendor"/>
    <property name="path.packages" value="${project.basedir}/packages"/>
    <property name="path.framework" value="${path.packages}/framework"/>
    <property name="path.utils" value="${project.basedir}/utils"/>
    <property name="translations.dump.flags" value=""/>
    <property name="backend-api.install-sh.arg" value="monorepo"/>

```

Reply:
Simply moving them to monorepo would make the most sense.